api/src/feeds/impl/feeds_api_impl.py

```diff
@@ -226,22 +227,37 @@ def get_gtfs_rt_feeds(
         municipality: str,
     ) -> List[GtfsRTFeed]:
         """Get some (or all) GTFS Realtime feeds from the Mobility Database."""
+        entity_types_list = entity_types.split(",") if entity_types else None
+
+        # Validate entity types are in [vp, sa, tu]
```

Comment:
Hardcoding the entity types is a bit cringy, no?

Reply:
This is a good point; I'll add an enum. We should have it defined in the OpenApi schema, but I recall that we had challenges with the previous code generator.

Reply:
Done.